src/libexpr/nixexpr.hh

```diff
@@ -274,6 +314,11 @@ struct ExprAttrs : Expr
     ExprAttrs() : recursive(false) { };
     PosIdx getPos() const override { return pos; }
     COMMON_METHODS
+
+    std::shared_ptr<const StaticEnv> bindInheritSources(
+        EvalState & es, const std::shared_ptr<const StaticEnv> & env);
+    Env * buildInheritFromEnv(EvalState & state, Env & up);
+    void showBody(const SymbolTable & symbols, std::ostream & str) const;
```

Comment:
```suggestion
    void showBindings(const SymbolTable & symbols, std::ostream & str) const;
```

In a lambda, the body is the part after `:`.
In a `let`, I would the expect the body to be the part after `in`, but this is the part between `let` and `in`.